### DISCUSSION TITLE
Natural shaft vs an extract surface

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13520] [Q&A]
I’m seeing some unexpected results in my simulation and was wondering if anyone could shed some light on it. I’m modeling a fire in an apartment with a low-level opening to maintain the HRR, an open lobby and stair door, and a natural opening at the top of the stair for inlet air. The main difference between two scenarios is that one has an extract surface while the other uses a natural shaft. Strangely, the natural shaft scenario results in higher airspeeds, which I wasn’t expecting based on previous cases. Also the natural scenario protects the stair from smoke. Any thoughts on why this might be happening?

![{C7A41557-8A79-4F16-BE2B-26255B8C81EA}](https://github.com/user-attachments/assets/e3cf48ae-149c-498c-b191-2393668347e4)
![{B828C714-FC62-4350-87A1-CAA7DEF8E2EE}](https://github.com/user-attachments/assets/4192d46f-f892-4ef6-ace3-b40ede4c7326)

**Comment by drjfloyd** [comment]
No idea what these images are showning in terms of quantity or which image is which case. No idea about the shaft dimensions, height, fire size, or any specific details on your simuilation.  About all that can be said with the information you have provided is if you had set an extract speed of 1 um/s would you be surprised if the natural case flowed more air? i.e, your assumptions about how much air can flow up the shaft due to natural convection may be incorrect.

**Comment by ShehabGamrah** [comment]
This compares velocity at a scale of 2m/s for both. The shaft and stair height is above 100m. The shaft is 1.5m2 cross sectional area. The extract was set at a high speed of 5m3/s. It's confusing how the natural shaft can create higher velocities even though the exhaust speed is high.

**Comment by ShehabGamrah** [comment]
Also why does smoke prefer to enter the natural shaft rather than the stair.

**Comment by drjfloyd** [comment]
An F5 tornado has wind speeds over 100 m/s.  This is entirely due to natural ventilation. Naturaly ventilation can generate high speeds. 
5 m3/s in a 1.5 m2 shaft is 3.3 m/s. That is not a particularly high speed.  

The speciifc air flow you see is a result of the specific geomteric details and fire scenario that you have defined in your input files the contents of which no one but you knows.

**Reply by ShehabGamrah** [reply]
Just to clarify, the natural shaft scenario doesn’t have an exhaust vent, while the other scenario does have one specified in the corridor. Based on previous simulations, the exhaust setup typically produces higher airspeeds in the stairwell and clears smoke faster. However, in this case, the natural shaft is performing better in terms of stair protection. One unusual aspect is that the building is quite tall, which might be amplifying the stack effect. I haven’t factored in wind or temperature differences at the building's 100+ meter height, so I’m curious—if the height is boosting natural ventilation, how exactly is it influencing this and how is the stack effect occurring? Also why does smoke prefer to enter the natural shaft and exit through the shaft rather than preferring to the head of stair opening.

**Reply by drjfloyd** [reply]
How can anyone possibly tell you why your specific simulaitons aren't performing as you expect when only you know the exact details of the inputs? 

The are many references out there that explain how to estimate pressure drops and flows due to stack effect
